pkg/store/bucket.go

```diff
@@ -2216,11 +2241,17 @@ func decodeSeriesForTime(b []byte, lset *[]symbolizedLabel, chks *[]chunks.Meta,
 	return len(*chks) > 0, d.Err()
 }
 
+type preloadIdx struct {
+	off uint32
+	i   int
```

Comment:
Can we have more meaningful variable name here?  :hug

Reply:
Done.